pkg/systemd/overview.jsx

```diff
@@ -38,8 +39,116 @@ import { MotdCard } from './overview-cards/motdCard.jsx';
 import { UsageCard } from './overview-cards/usageCard.jsx';
 import { ServerTime } from './overview-cards/serverTime.js';
 
+import moment from "moment";
+
 const _ = cockpit.gettext;
 
+class LoginMessages extends React.Component {
+    constructor() {
+        super();
+        this.state = { messages: {} };
+        this.permission = cockpit.permission({ admin: true });
+
+        this.close = this.close.bind(this);
+        this.goToLogs = this.goToLogs.bind(this);
+    }
+
+    componentDidMount() {
+        const bridge = cockpit.dbus(null, { bus: "internal" });
+        bridge.call("/LoginMessages", "cockpit.LoginMessages", "Get", [])
+                .done(text => {
+                    const obj = JSON.parse(text);
+                    if (obj.version == 1) {
+                        if (obj['fail-count'] > 5) {
+                            obj.type = "danger";
+                        } else if (obj['fail-count'] > 0) {
+                            obj.type = "warning";
+                        } else {
+                            obj.type = "info";
+                        }
+
+                        this.setState({ messages: obj });
+                    }
```

Comment:
please put a console.log() here with "unknown version blablah", so that this is easier to debug in the future.